app/conversations/tests/test_conversations_routes.py

```diff
@@ -18,6 +18,15 @@
 RANDOM_CONVERSATION_STATE = random.choice(list([s.value for s in ConversationState]))
 
 
+def is_uuid(string):
+    try:
+        uuid.UUID(string)
+        ok = True
+    except ValueError:
+        ok = False
+    return ok
+
+
 @pytest.mark.skip("FIXME")
```

Comment:
Will you be able to fix this test or remove it if it became redundant?

Reply:
Switched to use of new app.common.uuid.to_uuid function.